docs/source/guide/webhook_example.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Start model training with webhook events
+short: Webhook Example
+type: guide
+order: 651
+meta_title: Start model training with webhook events in Label Studio
+meta_description: Label Studio example documentation for starting machine learning model training in your MLOps pipeline in response to webhook events sent from Label Studio. 
+---
+
+
+
+
+## Example: Start model training using webhook events
```

Comment:
This will mostly just cover adding a specific type of webhook via the UI and maybe some other stuff too at the beginning and end.